compiler/parser/Makefile

```
@@ -7,10 +7,12 @@ ROLLUP_VERSION = 2.23.1
 deps = $(CURDIR)/deps
 npm = npm --global --prefix $(deps)
 
-all: zql.go zql.js zql.es.js
+PARSER_PREFIX = parser
 
-.PHONY: zql.go
-zql.go:
+all: $(PARSER_PREFIX).go $(PARSER_PREFIX).js $(PARSER_PREFIX).es.js
+
+.PHONY: $(PARSER_PREFIX).go
+$(PARSER_PREFIX).go:
```

Nit: I think this file would read better if you dropped PARSER_PREFIX and instead did s/zql/parser/g, but if you prefer to keep it, call it parser_prefix and put it next to deps and npm since, unlike the other all-caps variables, it isn't really a knob.
```suggestion
all: parser.go parser.js parser.es.js

.PHONY: parser.go
parser.go:
```